<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>O1_showall</class>
 <widget class="QDialog" name="O1_showall">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>882</width>
    <height>588</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_4">
   <item>
    <widget class="QLabel" name="label">
     <property name="font">
      <font>
       <family>Palatino Linotype</family>
       <pointsize>11</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>                                                                                             All Employee Information</string>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <widget class="QLabel" name="label_2">
         <property name="text">
          <string>                     Permanent Employees</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QTableView" name="permanent_tableView"/>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <widget class="QLabel" name="label_3">
         <property name="text">
          <string>                       Contract Employees</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QTableView" name="contract_tableView"/>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_3">
       <item>
        <widget class="QLabel" name="label_4">
         <property name="text">
          <string>                      Sales Employees</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QTableView" name="sales_tableView"/>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QPushButton" name="showall_loadButton">
       <property name="font">
        <font>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>LOAD</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="showall_closeButton">
       <property name="font">
        <font>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>Close</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QLabel" name="showall_connLabel">
     <property name="font">
      <font>
       <italic>true</italic>
      </font>
     </property>
     <property name="text">
      <string>[+] Connection Status</string>
     </property>
    </widget>
   </item>
  </layout>
  <zorder>showall_connLabel</zorder>
  <zorder>permanent_tableView</zorder>
  <zorder>contract_tableView</zorder>
  <zorder>sales_tableView</zorder>
  <zorder>label</zorder>
  <zorder>showall_loadButton</zorder>
  <zorder>showall_closeButton</zorder>
  <zorder>showall_closeButton</zorder>
  <zorder>label_2</zorder>
  <zorder>label_3</zorder>
  <zorder>label_4</zorder>
 </widget>
 <resources/>
 <connections/>
</ui>
